dexter/src/test/java/com/karumi/dexter/DexterInstanceTest.java

```diff
@@ -44,6 +44,7 @@
 @RunWith(MockitoJUnitRunner.class) public class DexterInstanceTest {
 
   private static final String ANY_PERMISSION = "noissimrep yna";
+  private static final Thread THREAD_SPEC = new TestThread();
```

Comment:
We should rename this constant to just ``THREAD`` the ``spec`` naming has been removed.